requirements.txt

```diff
@@ -1,25 +1,21 @@
-grpcio==1.25.0
-grpcio-tools==1.25.0
-protobuf==3.7.0
-aio-pika==3.0.1
-aiohttp==3.4.4
-coloredlogs==10.0
+grpcio==1.27.2
+grpcio-tools==1.27.2
+protobuf==3.11.3
+aiohttp==3.6.2
+coloredlogs==14.0
 earlgrey>=0.0.4
-fluent-logger==0.9.3
+fluent-logger==0.9.5
 jsonrpcclient==3.3.5
 jsonrpcserver==4.1.2
 leveldb==0.20
 plyvel>=1.0.5
-pika==0.12.0
-python-pkcs11==0.5.0
-requests==2.20.0
-cryptography==2.3.1
-sanic==18.12.0
+requests==2.23.0
+cryptography==2.8
 coincurve==13.0.0
 setproctitle==1.1.10
 transitions==0.6.8
 verboselogs==1.7
-websockets==6.0
-yappi==0.98
+websockets==8.1
+yappi==1.2.3
 eth-keyfile==0.5.1
 dataclasses; python_version < '3.7'
```

Comment:
This requirement is no more need.

Reply:
Your review intends that we "remove" those dependencies, not to "keep" current version of them. Right?

> About websockets
I tried to remove `websockets` at first.
But there's a explicit usage of it in Loopchain, and it is installed as a dependency of dependency.

> About yappi
I think removal of `yappi` will raise error if someone runs Loopchain by `$ loop [...]`. (See `./loopchain/__main__.py`

------
Unless we fix some code lines, I think removal of those dependencies could raise Exception.

Reply:
I mean `dataclasses; python_version < '3.7'`.

Reply:
Oh, apology.

[dataclasses](https://docs.python.org/3.7/library/dataclasses.html) is New in version 3.7!

I see. 

Thank you.

Reply:
But I found that python version under 3.7 is still able to install Loopchain.
I think we must enforce python version by using `python_requires` in `setup.py` or whatever.

Reply:
loopchain support 3.7.x only in [README](https://github.com/icon-project/loopchain#requirements).

Reply:
README says do not install python<3.7 but, I mean basically it is not prohibited.